src/test/java/BasePage.java

```diff
@@ -0,0 +1,28 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.time.Duration;
+
+public class BasePage {
+    WebDriver driver;
+    WebDriverWait wait;
+    Actions actions;
+
+    public BasePage (WebDriver givenDriver) {
+        driver = givenDriver;
+        wait = new WebDriverWait(driver, Duration.ofSeconds(5));
+        actions = new Actions(driver);
+    }
+
+    public WebElement findElement(By locator) {
+        return wait.until(ExpectedConditions.invisibilityOfElementLocated(locator));
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://github.com/Test-Pro-Bootcamp/A47-QA-Automation-Framework/assets/104761482/02a4645b-e712-4594-8e4a-f0c0c3a0e843)

HINT

```
public WebElement findElement(By locator) {
    return wait.until(ExpectedConditions.visibilityOfElementLocated(locator));
}
```